Jenkins build failing

Please look at [console output](https://openhab.ci.cloudbees.com/job/openHAB/lastFailedBuild/console) for broken OH1 build.  A number of OH2 features seem to be pulled into OH1 during the build now, and it fails.  Why does the OH1 build need to pull in Karaf, for example?

Builds fail locally and on Jenkins now. 

@kaikreuzer @teichsta

After #3627, build still fails locally and on Jenkins.  Local failure end:
```
[...]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 12:57 min
[INFO] Finished at: 2015-12-30T10:16:02+00:00
[INFO] Final Memory: 639M/1278M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.karaf.tooling:karaf-maven-plugin:4.0.3:verify (verify) on project openhab-addons-verify: Verification failures -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :openhab-addons-verify
```

--- a/features/pom.xml
+++ b/features/pom.xml
@@ -27,7 +27,7 @@
     <module>org.openhab.runtime.feature</module>
     <module>openhab-addons</module>
     <module>openhab-addons-external</module>
-    <module>openhab-addons-verify</module>
+<!--    <module>openhab-addons-verify</module>-->
   </modules>
 
     <build>

